packages/auth-providers/dbAuth/setup/src/__tests__/setup.test.ts

```diff
@@ -57,7 +57,9 @@ jest.mock('@redwoodjs/cli-helpers', () => {
     // jest.requireActual(@redwoodjs/cli-helpers) here, but I couldn't because
     // jest doesn't support ESM
     standardAuthHandler: async (args: AuthHandlerArgs) => {
-      args.notes && console.log(`\n   ${args.notes.join('\n   ')}\n`)
+      if (args.notes) {
+        console.log(`\n   ${args.notes.join('\n   ')}\n`)
+      }
```

Comment:
https://typescript-eslint.io/rules/no-unused-expressions: added as part of the `stylistic` preset. For consistency, it can enforce using `if (...) { ... }` instead of `... && ...`.

Reply:
I like this!